e2e-tests/commands/insurance/complete-alternative-currency-form.js

```diff
@@ -0,0 +1,41 @@
+import { NON_STANDARD_CURRENCY_CODE } from '../../fixtures/currencies';
+import { radios } from '../../pages/shared';
+import partials from '../../partials';
+import { INSURANCE_FIELD_IDS } from '../../constants/field-ids/insurance';
+
+const { CURRENCY_CODE, ALTERNATIVE_CURRENCY_CODE } = INSURANCE_FIELD_IDS.CURRENCY;
+
+/**
+ * completeAlternativeCurrencyForm
+ * presses hyperlink and completes the alternative currency form
+ * if isoCode provided - clicks the radio for currency and submits alternative currency form
+ * if alternativeCurrency true, then enters alternative currency and submits alternative currency form
+ * @param {String} isoCode - isoCode provided for radio selection
+ * @param {Boolean} alternativeCurrency - if alternative currency should be entered
+ */
+const completeAlternativeCurrencyForm = ({ isoCode, alternativeCurrency, linkFieldId }) => {
```

Comment:
`completeAndSubmit` ? It has submit button clicks

Maybe not for now, and not sure if this would be useful for this form, but in others we have `completeXForm` and `completeAndSubmit` so we can complete the form without submitting

Reply:
don't think it'll be useful in this form

Reply:
Ok can we rename to `completeAndSubmit ...` though because it reads like it's not submitting it, all other commands have "andSubmit" if it's submitting